### ISSUE TITLE
Ensure type names are generated uniformly independent of capitalization

### ISSUE THREAD

**Issue by tnull** [NordSecurity/uniffi-bindgen-cs#28]
I have a type declared as `interface ASDFObject` in my UDL file, which works without issue in Swift/Kotlin/Python.

When trying to generate CS bindings, the output file however includes

```csharp
public class AsdfObjectSafeHandle: FFISafeHandle {
```

while the constructors then are defined as

```csharp
public static extern ASDFObjectSafeHandle ... 
```

... which of course doesn't compile. If I switch the Rust/UDL type name to be `AsdfObject` everything seems to be generated uniformly, which I however would like to avoid.

**Comment by arg0d** [comment]
Good catch. Is the expected class name for your C# bindings `class AsdfObject`? Or should it be `class ASDFObject`?

**Comment by dfetti** [comment]
Fixed in #103
